### PR TITLE
Add new parameter to overwrite default adaptive card json object

### PR DESCRIPTION
**Pull request by nikstuckenbrock**
**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added a new parameter to provide a customized adaptive card json object for more flexibility if needed.

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by thefrieddan1** [comment]
Hi @nikstuckenbrock 
Please share a video of the contribution.
Can you also please share what is your motivation by adding it.

**Comment by nikstuckenbrock** [comment]
Hi @thefrieddan1,

you can find demo [here](https://drive.google.com/file/d/1yoj_4O8MzpOVaVvkkPsVRWZgca1Zy3jX/view?usp=sharing). The last update introduced a new feature that sends an adaptive card JSON object to a webhook instead of the normal payload. This is/was necessary because Microsoft wants to replace the Teams internal webhooks with so-called workflows. However, these workflows require a different format: Adaptive Card JSON. However, there are currently still a few bugs when rendering Markdown, for example. In order to still have the possibility to define your own [Adaptive Cards](https://adaptivecards.io/designer/), I have now added a parameter that allows you to overwrite the JSNON object if necessary.

**Comment by thefrieddan1** [comment]
@nikstuckenbrock we have another contribution regarding the adaptive cards and the move to workflows.
I need to understand the other contribution as well and see how both contributions fit in.
Thanks

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeamsWebhook/MicrosoftTeamsWebhook.yml [hunk 1]
```diff
@@ -48,9 +48,12 @@ script:
       - 'true'
       - 'false'
       defaultValue: true
+    - description: JSON object overwriting the default adaptive card JSON.
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeamsWebhook/MicrosoftTeamsWebhook.yml:51]:
```suggestion
    - description: JSON object used to overwrite the default adaptive card JSON.
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeamsWebhook/README.md [hunk 2]
```diff
@@ -29,11 +29,12 @@ Send a message to Microsoft Teams via Incoming Webhook.
...
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| message | The message to send.  For example: "This is a message from Cortex XSOAR,". Default is None. | Optional | 
-| team_webhook | The alternative webhook for a different team.  If not defined, the integration's default webhook is used. | Optional | 
-| alternative_url | The alternative URL to send in place of the link to the XSOAR Investigation. | Optional | 
-| url_title | The title for the link, defaults to "Cortex XSOAR URL". Default is Cortex XSOAR URL. | Optional | 
-| adaptive_cards_format | Should the adaptive card format be used or a single text message. | Optional | 
+| message | The message to send.  For example: "This is a message from Cortex XSOAR,". Default is None. | Optional |
+| team_webhook | The alternative webhook for a different team.  If not defined, the integration's default webhook is used. | Optional |
+| alternative_url | The alternative URL to send in place of the link to the XSOAR Investigation. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeamsWebhook/README.md:34]:
```suggestion
| alternative_url | The alternative URL to send in place of the link to the Cortex XSOAR Investigation. | Optional |
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeamsWebhook/README.md [hunk 3]
```diff
@@ -29,11 +29,12 @@ Send a message to Microsoft Teams via Incoming Webhook.
...
+| url_title | The title for the link, defaults to "Cortex XSOAR URL". Default is Cortex XSOAR URL. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeamsWebhook/README.md:35]:
```suggestion
| url_title | The title for the link,. Default is Cortex XSOAR URL. | Optional |
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeamsWebhook/README.md [hunk 4]
```diff
@@ -29,11 +29,12 @@ Send a message to Microsoft Teams via Incoming Webhook.
...
+| adaptive_cards_format | Should the adaptive card format be used or a single text message. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeamsWebhook/README.md:36]:
```suggestion
| adaptive_cards_format | Whether the adaptive card format be used or a single text message. | Optional |
```

File: Packs/MicrosoftTeams/Integrations/MicrosoftTeamsWebhook/README.md [hunk 5]
```diff
@@ -29,11 +29,12 @@ Send a message to Microsoft Teams via Incoming Webhook.
...
+| overwrite_adaptive_card_json | JSON object overwriting the default adaptive card JSON. | Optional |
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/Integrations/MicrosoftTeamsWebhook/README.md:37]:
```suggestion
| overwrite_adaptive_card_json | JSON object used to overwrite the default adaptive card JSON. | Optional |
```

File: Packs/MicrosoftTeams/ReleaseNotes/1_5_3.md [hunk 6]
```diff
@@ -0,0 +1,6 @@
+#### Integrations
+
+##### Microsoft Teams via Webhook
+- Updated the Docker image to: *demisto/python3:3.11.10.111526*.
+
+- Added a argument (overwrite_adaptive_card_json) to ms-teams-message that enables customers to overwrite the default adaptive card JSON object with custom layouts and actions using the [Adaptive Card designer](https://adaptivecards.io/designer/).
```
Comment (ShirleyDenkberg) [inline Packs/MicrosoftTeams/ReleaseNotes/1_5_3.md:6]:
```suggestion
- Added the *overwrite_adaptive_card_json* argument to the **ms-teams-message** that enables customers to overwrite the default adaptive card JSON object with custom layouts and actions using the [Adaptive Card designer](https://adaptivecards.io/designer/).
```

**Comment by ShirleyDenkberg** [comment]
@thefrieddan1 Doc review completed.

**Review by thefrieddan1 — APPROVED** [review]

**Pull request merged**